src/schoology-extractor/schoology_extractor/api/request_client.py

```diff
@@ -164,7 +164,7 @@ def get_assignments_by_section_ids(
                 self,
                 page_size,
                 self.get(url),
-                "assignment",
+                RESOURCE_NAMES.ASSIGNMENT,
```

Comment:
Generally I have pulled strings like this into a constant if using in multiple places and change could happen. Makes me wonder if the UDM mappers should have more column names as constants. Worth thinking about - we might want to go ahead and change those mappers to use constants in more places.